CHANGELOG.md

```diff
@@ -1,3 +1,139 @@
+### citus v10.0 (February 16, 2021) ###
+
```

Comment:
I think we should have the most interesting things at the top:
1. columnar
2. rebalancer
3. Enables Single-node Citus for production workloads (add this as an item as it is not explicitly mentioned in others)
4. Local table distributed table joins
5. Foreign key to/from local tables